xml/System.Threading/ThreadPool.xml

```diff
@@ -1267,13 +1268,13 @@
         <Parameter Name="preferLocal" Type="System.Boolean" Index="2" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TState">To be added.</typeparam>
-        <param name="callBack">To be added.</param>
-        <param name="state">To be added.</param>
-        <param name="preferLocal">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TState">The type of elements of <paramref name="state" />.</typeparam>
+        <param name="callBack">An <see cref="T:System.Action`1" /> representing the method to execute.</param>
+        <param name="state">An object containing data to be used by the method.</param>
+        <param name="preferLocal">Unlike other QueueUserWorkItem methods, prefer exeucting in the local thread pool and not the global thread pool.</param>
```

Comment:
```suggestion
        <param name="preferLocal">`true` to execute in the local threadpool; `false` to execute in the global thread pool.</param>
```
The comment about other UnsafeQueueUserWorkItem methods can go in the Remarks section.